manifests/init.pp

```diff
@@ -94,7 +274,6 @@
   }
   validate_bool($service_enable_real)
 
-  validate_absolute_path($logfile)
```

Comment:
went down to [line 523 ](https://github.com/ghoneycutt/puppet-module-nscd/pull/36/files#diff-60ae41fd0a31977447947f59940ee9a4R523)